path.cabal

```diff
@@ -21,49 +21,49 @@ library
   hs-source-dirs:    src/
   ghc-options:       -Wall -O2
   exposed-modules:   Path, Path.Internal
-  build-depends:     base >= 4 && <5
-                   , exceptions
-                   , filepath < 1.2.0.1 || >= 1.3
-                   , template-haskell
+  build-depends:     aeson
+                   , base       >= 4.7     && < 5.0
                    , deepseq
-                   , aeson
-                   , hashable >= 1.2 && < 1.3
+                   , exceptions >= 0.6     && < 0.9
+                   , filepath   < 1.2.0.1  || >= 1.3
+                   , hashable   >= 1.2     && < 1.3
```

Comment:
> I'd prefer to keep them until we are forced to relax them :-D

I strongly disagree but would like to hear what everyone else thinks about this.

> About bounds on test suite dependencies: why not document verisons that the test suite is supposed to be used with...

Compatible versions are already implicitly documented in the stack resolvers used by the current .travis.yml and the Stackage snapshots that path is released with.

In my experience bounds on test suites mostly create needless maintenance work for package maintainers and stackage maintainers. But maybe you have other experiences?